### ISSUE TITLE
[RFC7234] Cache warnings

### ISSUE THREAD

**Issue by krizhanovsky** [tempesta-tech/tempesta#1]
Tempesta MUST attach 110 warning header on [sending stale responses](https://tools.ietf.org/html/rfc7234#section-4.2.4).

Depends on #513 (cache entry freshness calculation).

**Comment by krizhanovsky** [comment]
RFC 7234 has reduced set of cases when warning MUST be generated, in particular only 214 - "Transformation Applied" (5.5.6). Currently we do not do any response transformations, so postpone the task to later phases.
